src/test/java/com/spotify/github/v3/clients/RepositoryClientTest.java

```diff
@@ -171,6 +174,18 @@ public void listCommits() throws Exception {
         commits.get(0).commit().tree().sha(), is("6dcb09b5b57875f334f61aebed695e2e4193db5e"));
   }
 
+  @Test
+  public void listPullRequestsForCommit() throws Exception {
+    final CompletableFuture<List<PullRequestItem>> fixture =
+        completedFuture(
+            json.fromJson("[" + getFixture("../prs/pull_request_item.json") + "]", LIST_PR_TYPE_REFERENCE));
+    when(github.request(eq("/repos/someowner/somerepo/commits/thesha/pulls"), eq(LIST_PR_TYPE_REFERENCE), any()))
+        .thenReturn(fixture);
+    final List<PullRequestItem> commits = repoClient.listPullRequestsForCommit("thesha").get();
```

Comment:
Does it return commits? Or pull requests? if the latter is true, should be renamed to `pullRequests`? (as list)

Reply:
fixed